native/src/components/News.tsx

```diff
@@ -81,7 +81,7 @@ const News = (props: PropsType): ReactElement => {
   }
 
   if (loading) {
-    return <LoadingSpinner />
+    return <LoadingSpinner testID='loadingSpinner' />
```

Comment:
maybe we can set the `testID directly in loading spinner, as this could be useful for other tests

Reply:
Thought about that, yes. However, the loading spinner is currently just a simple styled component and we'd need to add a wrapper. I'd figured that that is not worth it. Open to discuss that though.

Reply:
Hmm. If that's the case then it's probably okay. 👍🏼